runbotics/runbotics-desktop/src/core/websocket/cache/id-name-cache.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class IdNameCacheService {
```

Comment:
this class is not specific in any way so imo it's name should stay general like `StorageService` which in fact is already in the config module but I like your implementation much more.

Reply:
imo the best way to do it its like you have a single implementation of generic storage service and when you need something specific then you extend it with proper typings. Then you can name it more specific